spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.gcp.spanner.instance-id=integration-instance
+spring.cloud.gcp.spanner.database=spanner-sample-test-db
```

Comment:
I think you might get an error because our draconian checkstyle file requires newlines at the end of files